docs/src/modules/components/ChartFeaturesGrid.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import Grid from '@mui/material/Unstable_Grid2';
+import Grid from '@mui/material/Grid';
```

Comment:
Avoiding the use of `Grid2`, because of breaking changes between v5 and v6.
Once we make v6 the default, we can change all usages of `Grid` to `Grid2`. 👌

Reply:
Because it introduces problems when testing/running our codebase on v5 and v6 of `@mui/material`.
There is no graceful migration for `Unstable_Grid2`, it only works with one props on v5 and different props on v6.
We can use `Grid2` when we change the default of `@mui/material` to v6 in our repo (possibly when we start work on v8). 😉